app/workers/geocode_application_address_worker.rb

```diff
@@ -8,23 +8,10 @@ def perform(application_form_id)
 
```

Comment:
Maybe we should reduce the retry count here as well? Down to 5 perhaps?

Reply:
agreed, I think even 5 is too many, generally if these things fail, they fail forever.